src/main/java/us/kilroyrobotics/subsystems/CoralIntake.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package us.kilroyrobotics.subsystems;
+
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.SparkMax;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import us.kilroyrobotics.Constants.CoralMechanismConstants;
+
+public class CoralIntake extends SubsystemBase {
+    /** Creates a new Intake. */
+    private SparkMax wheelMotor;
```

Comment:
We should come up with a better name for this object

Reply:
What did you have in mind?

Reply:
intakeMotor is a better name tbh. And then maybe have a comment like "motor connected to compliant wheel in coral mechanism"

Reply:
we plan to also have an algae intake motor, so it would still need to include coral somewhere in it

Reply:
I can name it CoralIntakeMotor